tools/debian/rules

```diff
@@ -8,6 +8,12 @@ ifneq ($(shell dpkg -s libpcp3-dev >/dev/null 2>&1 && echo yes),yes)
 	CONFIG_OPTIONS = --disable-pcp
 endif
 
+# PackageKit crashes on update information on Ubuntu 16.04, which makes
+# "Software Updates" useless (LP: #1689820)
```

Comment:
This is not a reason not to ship the feature. That's a bug in Ubuntu. And we should track it as a known naughty issue.